rhoas-cli-service-registry/README.adoc

```diff
@@ -97,6 +97,10 @@ This example creates a {registry} instance called `my-registry`.
 ----
 $ rhoas service-registry create --name my-registry
 Creating {registry} instance with name: my-registry
```

Comment:
Line 99/99: This line is part of the command output so should be moved to become line 103.

Reply:
I've removed the command output entirely, so this is no longer there.